acceptance/tests/external_ca_support/jetty_external_root_ca.rb

```diff
@@ -93,19 +89,21 @@
 create_remote_file master, "#{testdir}/etc/agent/ssl/certs/#{fixtures.agent_name}.pem", fixtures.agent_cert
 create_remote_file master, "#{testdir}/etc/agent/ssl/private_keys/#{fixtures.agent_name}.pem", fixtures.agent_key
 
-on master, "cp -p '#{jetty_confdir}/webserver.conf' '#{testdir}/webserver.conf.orig'"
 create_remote_file master, "#{jetty_confdir}/webserver.conf",
                    fixtures.jetty_webserver_conf_for_trustworthy_master
 
 master_opts = {
     'master' => {
-        'ca' => false,
         'certname' => fixtures.master_name,
         'ssl_client_header' => "HTTP_X_CLIENT_DN",
         'ssl_client_verify_header' => "HTTP_X_CLIENT_VERIFY"
     }
 }
 
+# disable CA service
+# https://github.com/puppetlabs/puppetserver/blob/master/documentation/configuration.markdown#service-bootstrapping
+create_remote_file master, "#{jetty_confdir}/../services.d/ca.conf", "puppetlabs.services.ca.certificate-authority-disabled-service/certificate-authority-disabled-service"
```

Comment:
I think this file needs to have a .cfg file extension, like ca.cfg, and not ca.conf.  I don't think the Trapperkeeper bootstrapping logic will load a file that has any file extension other than .cfg.